### ISSUE TITLE
import/named issue in 2.0.0

### ISSUE THREAD

**Issue by dontsave** [typescript-eslint/typescript-eslint#920]
It seems like there is an issue with eslint rule `import/named`, in which typescript-eslint errors on importing perfectly fine named exports

possibly related: https://github.com/benmosher/eslint-plugin-import/issues/1282#issuecomment-525772853

**What code were you trying to parse?**

```ts
import { ThemeProps } from 'styled-components'
```

**What did you expect to happen?**
no error

**What actually happened?**
`ThemeProps not found in 'styled-components'eslint(import/named)`

**Versions**
2.0.0

**Comment by DonGissel** [comment]
THANK you! This issue has been driving me nuts over the last hour or so!

**Comment by JamesHenry** [comment]
You say `typescript-eslint` errors, but that is not accurate - your are getting a lint error from `import/named`, which is coming from the plugin you link to.

I'm not sure what the expectation is here - we cannot do anything to influence how separate plugins work, so it seems like this is an issue that needs to be tackled on the plugin side?

If a plugin is reporting a lint error on code, and you don't feel it should, then that has to be looked in to within the plugin...

Let me know if there is any more information we are missing here

**Comment by dontsave** [comment]
Ah yeah I was hasty to report this, not realizing that repo was a plugin of eslint as opposed to eslint itself. I'll follow up on that thread.
